sound/hybrid_font.h

```diff
@@ -138,25 +138,46 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
-  
-  void StartSwing() override {
-    if (!guess_monophonic_) {
-      if (swing_player_) {
-        // avoid overlapping swings, based on value set in ProffieOSSwingOverlap.  Value is
-        // between 0 (full overlap) and 1.0 (no overlap)
-        if (swing_player_->pos() / swing_player_->length() >= config_.ProffieOSSwingOverlap) {
-          swing_player_->set_fade_time(swing_player_->length() - swing_player_->pos());
-          swing_player_->FadeAndStop();
-          swing_player_.Free();
-          swing_player_ = PlayPolyphonic(&swng);
+
+  void StartSwing(const Vec3& gyro, float swingThreshold_, float slashThreshold_) override {
+    Vec3 gyro_slope = fusor.gyro_slope();
+    float rss = sqrtf(gyro_slope.z * gyro_slope.z + gyro_slope.y * gyro_slope.y)/57.3;
```

Comment:
add a comment like // radians per second per second
Also, instead of /57.3, do: * (M_PI / 180.0);
It's more clear, and potentially faster (multiplication is a little simpler than division).

Reply:
ok cool, will get that changed.

Reply:
Resolved.